build.gradle

```diff
@@ -9,7 +9,7 @@ buildscript {
         classpath 'com.palantir.gradle.jdks:gradle-jdks:0.69.0'
         classpath 'com.palantir.gradle.jdkslatest:gradle-jdks-latest:0.21.0'
         classpath 'com.palantir.gradle.plugintesting:gradle-plugin-testing:0.11.0'
-        classpath 'com.palantir.gradle.externalpublish:gradle-external-publish-plugin:1.22.0'
+        classpath 'com.palantir.gradle.externalpublish:gradle-external-publish-plugin:1.21.0'
```

Comment:
We need to use an old version of the publish plugin so we can actually publish this